packages/crusher-electron-app/src/extension/v2/ui/components/Sidebar/Steps.tsx

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import { Text } from "@dyson/components/atoms/text/Text";
+import { Checkbox, CheckboxProps } from "@dyson/components/atoms/checkbox/checkbox";
+import { css } from "@emotion/react";
+import { MoreIcon } from "crusher-electron-app/src/extension/assets/icons";
+import { TextBlock } from "@dyson/components/atoms/textBlock/TextBlock";
+import { Conditional } from "@dyson/components/layouts";
+import { Dropdown } from "@dyson/components/molecules/Dropdown";
+
+export const containerStyle = css`
+	padding: 26rem;
+`;
+
+export function Steps(): JSX.Element {
+	const [s, setS] = React.useState([]);
+	const [selected, setSelected] = React.useState(new Set());
+
+	const toggleStep = React.useCallback(
+		(index) => {
+			const selectedSteps = new Set(selected);
```

Comment:
Using index as a unique key is usually a bad idea, let's use a map here with a unique that out recorder will assign each step.

Reply:
This is only for implementation, we can do it later on